usrp3/tools/scripts/viv_sim_project.tcl

```diff
@@ -60,12 +60,20 @@ foreach src_file $design_srcs {
     } elseif [expr [lsearch {.ngc .edif} $src_ext] >= 0] {
         puts "BUILDER: Adding Netlist : $src_file"
         read_edif $src_file
+    } elseif [expr [lsearch {.dat} $src_ext] >= 0] {
+        puts "BUILDER: Adding Data File : $src_file"
+        add_files -fileset $sim_fileset -norecurse $src_file
```

Comment:
Is it needed to add .dat files from $design_sources and add them to $sim_fileset? If they are for simulation only then shouldn't these have been included as SIM_SRCS in a Makefile rather than as DESIGN_SRCS?